docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -15,12 +15,13 @@ which edx-proctoring can call directly since it is a plugin that is installed di
 
 Decision
 --------
-We've decided to use an event driven architecture because unlike the legacy system (edx-porctoring), edx-exams is an independent service.
+We've decided to use an event driven architecture because unlike the legacy system (edx-proctoring), edx-exams is an independent service.
 As such, we plan to use the event bus to send info to edx-platform services without needing a response as one would in a REST framework.
 
-Below are lists of downstream effects of exam submission and review that we will or will not be translating into events in edx-exams as part of this decision.
+Below are lists of downstream effects of exam submission and review that we will or will not be impementing as downstream effects that will be
+triggered by events emitted in edx-exams per this decision.
 
-Downstream effects to be implemented as events:
+Downstream effects to be implemented:
```

Comment:
I'm noticing a slight mismatch between the names in this section and the names in the `Downstream effects to be implemented as part of this decision` section.